lang/polyml/Portfile

```diff
@@ -1,27 +1,25 @@
 PortSystem 1.0
+PortGroup           github 1.0
 
-name                polyml
-version             5.5.2
+github.setup        polyml polyml 5.7.1 v
 categories          lang ml
```

Comment:
I agree, "ml" is not particularly meaningful.  However, I think there should be a tag for the language generally known as SML.  There is only one dialect though there have been two revisions and the most recent is known as [SML97](https://github.com/SMLFamily/The-Definition-of-Standard-ML-Revised).  That tag should also be used by:
- [lang/mosml](https://github.com/macports/macports-ports/tree/master/lang/mosml)
- [lang/mosml-dynlibs](https://github.com/macports/macports-ports/tree/master/lang/mosml-dynlibs)
- [lang/smlnj](https://github.com/macports/macports-ports/tree/master/lang/smlnj)

Would it make sense to change these occurrences of "ml" en masse in a separate commit?  I suggest "ml" is changed to "sml".  Also, [lang/sml-mode.el](https://github.com/macports/macports-ports/tree/master/lang/sml-mode.el) should probably have the "sml" tag added.

Reply:
I think `sml` would be the correct category, yes. Anything still labeled `ml` that is actually `ocaml` should be moved to the `ocaml` category (which is large).

Reply:
BTW, just FYI, generally, you should let the person who made the comment resolve it rather than presuming it's resolved. In this case, of course, I agree.